README.md

```diff
@@ -41,39 +41,26 @@ __Remember: A butter knife is like [a dagger][1] only infinitely less sharp.__
 Download
 --------
 
-Configure your project-level `build.gradle` to include the 'android-apt' plugin:
+Configure your project-level `build.gradle` to use at least vesion 2.2.0 of the Android Gradle Plugin:
 
 ```groovy
 buildscript {
-  repositories {
-    mavenCentral()
-   }
-  dependencies {
-    classpath 'com.neenbedankt.gradle.plugins:android-apt:1.8'
-  }
+    ...
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.2.0'  // 2.2.0 or higher
```

Comment:
Comment is redundant with the sentence above and people will end up copy/pasting it into build files. Let's remove.